src/main/webapp/app/entities/budget/budget-create.tsx

```diff
@@ -0,0 +1,164 @@
+                    }}
+                  />
+                </AvGroup>
+                <AvGroup>
+                  <Label id="nameLabel" for="budget-name">
+                    <Translate contentKey="clubmanagementApp.eventBudget.name">Name</Translate>
+                  </Label>
+                  <AvField
+                    id="budget-name"
+                    type="text"
+                    name="name"
+                    required
+                    validate={{
+                      required: { value: true, errorMessage: 'Please Enter a name for this budget' },
+                      maxLength: { value: 100, errorMessage: 'Name cannot be more than 100 characters' }
+                    }}
+                  />
+                </AvGroup>
+                <AvGroup>
+                  <Label id="typeLabel" for="budget-type">
+                    <Translate contentKey="clubmanagementApp.eventBudget.type">Type</Translate>
+                  </Label>
+                  <AvInput id="budget-type" type="select" className="form-control" name="type">
+                    <option value="" selected disabled>
+                      {translate('global.select.selectOne')}
+                    </option>
+                    <option value="INCOME">{translate('clubmanagementApp.TransactionType.INCOME')}</option>
+                    <option value="EXPENSE">{translate('clubmanagementApp.TransactionType.EXPENSE')}</option>
+                  </AvInput>
+                </AvGroup>
+                <AvGroup>
+                  <Label id="detailsLabel" for="budget-details">
+                    <Translate contentKey="clubmanagementApp.eventBudget.details">Details</Translate>
+                  </Label>
+                  <AvInput
+                    id="budget-details"
+                    type="textarea"
+                    name="details"
+                    validate={{
+                      maxLength: { value: 200, errorMessage: 'Details cannot be more than 200 characters' }
+                    }}
+                  />
+                </AvGroup>
+                <span className="text-error">{errorMessage ? errorMessage.response.data.detail : ''}</span>
+                <div className="text-center mx-4 d-flex justify-content-between justify-content-md-center mb-2">
+                  <Button tag={Link} id="cancel-save" to={`/entity/event-budget/event/${eventId}`} replace color="cancel">
+                    <FontAwesomeIcon icon="arrow-left" />
+                    &nbsp;
+                    <Translate contentKey="entity.action.cancel">Cancel</Translate>
+                  </Button>
+                  &nbsp;
+                  <Button color="action" id="save-entity" type="submit" disabled={updating}>
+                    <FontAwesomeIcon icon="save" />
+                    &nbsp;
+                    <Translate contentKey="entity.action.create">Create</Translate>
```

Comment:
`entity.action.create` has not created in the `src/main/webapp/i18n/en/global.json` yet